.github/workflows/test.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         ipfs:
           - "0.10"
           - "0.11"
-          - "0.12"
+          - "0.12.1"
```

Comment:
Is this change required? I have added semver resolution in the corresponding workflow, so it should resolve to the latest point release without version pinning to the third point.

Reply:
Let me revert this to just use 0.12 here then inspect the Action.

Reply:
@ibnesayeed You were correct -- the 0.12 specification here was correctly resolved to 0.12.1.